src/OrchardCore.Modules/OrchardCore.ContentFields/Fields/UserPickerField.cs

```diff
@@ -0,0 +1,10 @@
+using System;
+using OrchardCore.ContentManagement;
+
+namespace OrchardCore.ContentFields.Fields
+{
+    public class UserPickerField : ContentField
+    {
+        public string[] UserIds { get; set; } = Array.Empty<string>();
```

Comment:
Should we store the user name here as well, possibly as a less well known property like tag names?

Or as a concrete value?

This would be necesary to get the value into the lucene index

Reply:
Ok to store usernames too.